### PR TITLE
Add ESPv2 deployment files

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

### CODE REVIEW

**Comment by qiwzhang** [comment]
Due to testing requirement of adding these deployment files into language specific repo.  I decided to add them here.  We have full control of this repo.   These deployment files are used in our public doc.  This will speed up our dock update for ESPv2.

Please look at new  folder structure in top level README file.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
 
 This repository contains samples and utilities for using Google Cloud Endpoints v2 on Compute Engine or Container Engine
+
+Folder structures:
+* gettting-started: ESPv2 GKE depolyment yaml files used Cloud Endpoint [tutorials](https://cloud.google.com/endpoints/docs/openapi/tutorials).
+* gke: ESPv2 deployment yaml files for GKE.
```
Comment (nareddyt) [inline README.md:7]: Nit: GKE -> Google Kubernetes Engine

No need to change folder name, just descriptions across README

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
```
Comment (TAOXUY) [inline README.md:1]: Are GKE/K8S called container engine? Sound not straightforward to me.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
 
 This repository contains samples and utilities for using Google Cloud Endpoints v2 on Compute Engine or Container Engine
+
+Folder structures:
+* gettting-started: ESPv2 GKE depolyment yaml files used Cloud Endpoint [tutorials](https://cloud.google.com/endpoints/docs/openapi/tutorials).
```
Comment (TAOXUY) [inline README.md:6]: add redirection for this folders?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
 
 This repository contains samples and utilities for using Google Cloud Endpoints v2 on Compute Engine or Container Engine
+
+Folder structures:
```
Comment (TAOXUY) [inline README.md:5]: The structure looks not good
https://screenshot.googleplex.com/AyFpdm4sxyj

File: getting-started/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+These GKE deployment files are used in Cloud Endpoints public [documents](https://cloud.google.com/endpoints/docs/openapi/tutorials). They are for ESPv2.
```
Comment (TAOXUY) [inline getting-started/README.md:2]: ditto 
https://screenshot.googleplex.com/x7D81Aq8h3H

File: README.md (hunk 2 above)
Comment (qiwzhang) [reply README.md:1]: Yes, GKE used to be "Google Container Engine", now it changed to "Google Kubernetes engine".  I will update that

File: README.md (hunk 3 above)
Comment (qiwzhang) [reply README.md:6]: Done

File: README.md [hunk 6]
```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
 
-This repository contains samples and utilities for using Google Cloud Endpoints v2 on Compute Engine or Container Engine
+This repository contains samples and utilities for using Google Cloud Endpoints on GCE(Google Compute Engine) or GKE(Google Kubernetes Engine)
+
+## Repository Structure:
+* [gettting-started](gettting-started): ESPv2 Google Kubernetes Engine depolyment yaml files used in Cloud Endpoint [tutorials](https://cloud.google.com/endpoints/docs/openapi/tutorials).
```
Comment (nareddyt) [inline README.md:6]: deployment

File: README.md [hunk 7]
```diff
@@ -1,3 +1,9 @@
...
+* [gke](gke): ESPv2 deployment yaml files for Google Kubernetes Engine.
+* [kubernetes](kubernetes): ESPv2 deployment yaml files for Kubernetes. It is almost the same as GKE except that they all need to mount a service account key file to te ESP container.
```
Comment (nareddyt) [inline README.md:8]: the

File: README.md (hunk 6 above)
Comment (qiwzhang) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (qiwzhang) [reply README.md]: Done

**Pull request merged**